README.md

```diff
@@ -1 +1,105 @@
-# drone-artifactory-docker-buildinfo
\ No newline at end of file
+# Artifactory Docker Build Info Plugin
+
+A Drone plugin that publishes Docker image build information to JFrog Artifactory, including build details, VCS information, and SHA256 verification.
+
+## Features
+
+- Publishes Docker image build information to JFrog Artifactory
+- Verifies image SHA256 hash
+- Adds Git commit information (VCS data) to build metadata
+- Creates and publishes detailed build information
+- Supports multiple authentication methods
+
+## Requirements
+
+- JFrog Artifactory instance
+- Valid authentication credentials (Access Token, API Key, or Username/Password)
+- Docker image published to Artifactory registry
+
+## Authentication
+
+The plugin supports three authentication methods:
+
+1. Access Token (Recommended)
+2. Username and Password
+3. API Key
+
+## Configuration
+
+### Required Settings
+
+| Setting | Description |
+|---------|-------------|
+| `url` | JFrog Artifactory URL |
+| `docker_image` | Full path to Docker image in Artifactory |
+| `build_name` | Name of the build |
+| `build_number` | Build number (usually pipeline sequence ID) |
+
+### Optional Settings
+
+| Setting | Description |
+|---------|-------------|
+| `access_token` | JFrog access token for authentication |
+| `username` | JFrog username (alternative to access token) |
+| `password` | JFrog password (alternative to access token) |
+| `api_key` | JFrog API key (alternative to access token) |
+| `build_url` | URL to the build in Harness CI |
+| `git_path` | Path to Git repository (defaults to workspace) |
```

Comment:
create a table similar to the README.md [here](https://github.com/harness-community/drone-gcp-oidc?tab=readme-ov-file#drone-gcp-oidc)

Reply:
done